pkg/runtime/reconciler_test.go

```diff
@@ -88,7 +346,7 @@ func TestReconcilerUpdate(t *testing.T) {
 	rd.On("Delta", desired, latest).Return(
 		delta,
 	)
-	rd.On("UpdateCRStatus", latest).Return(false, nil)
```

Comment:
funnily enough, UpdateCRStatus never returns false...

https://github.com/aws-controllers-k8s/code-generator/blob/c77aa9c75d944952dee198029ba9822691cd82b0/templates/pkg/resource/descriptor.go.tpl#L60-L68

It was always supposed to be a place where resource managers/descriptors would be able to place custom code that set various Status fields (Conditions, etc) but I never made use of the functionality.